README.md

```diff
@@ -111,15 +111,19 @@ connection_pool.close()
 You should make sure the scan client can connect to the address of storage which see from `SHOW HOSTS` 
 
 ```python
-from nebula2.mclient import MetaCache
```

Comment:
- As mentioned in the conversation, it's better not to recommend manually filling storage addresses over the auto-discovering via metaD, instead, if we introduce this ability(it means application/user has to discover/maintain this list themselves), we should provide it in examples as an Option-B(second option).

- Please add extra spaces after the comma:
`a, b` is better than `a,b`